examples/mysql-replication/master/2.2/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mysql:latest
+FROM mysql:8.2
```

Comment:
@jpechane  @MartinMedek How come that Debezium 2.2 (released in April and May 2023) can use MySQL 8.2 here, which was released at the end of October 2023. Debezium 2.3.3/2.4.x are the earliest versions that should use it, no?

Reply:
@rk3rn3r While technically corect I think we can keep it as is.

Reply:
What was the reason for this change in all those versions?

Reply:
I guess the Dockerfile was just copied over instead of checking which MySQL version is the maximum one tested with. I really don't think this is real issue. These examples images are just used for testing. I'd recommend to not spent any more time discussing it as we are now on 2.5 anyway.
Maybe we can just take a mental notice that in the next case we don't need to touch the older images at all as they are on obsolete branch and we are not bound to fix them.

Reply:
Well none of the images worked since 8.3 mysql release. In theory I could fix just the versions, that support mysql 8.2 and leave the rest broken (?). I didn't go though the older versions of Debezium and their supported mysql versions because this image is used just in system testsuite, that always uses latest version so additional testing of various mysql versions wasn't worth the effort.